src/wp-includes/user.php

```diff
@@ -4951,3 +4951,48 @@ function wp_is_application_passwords_available_for_user( $user ) {
 	 */
 	return apply_filters( 'wp_is_application_passwords_available_for_user', true, $user );
 }
+
+/**
+ * Register the user meta property for persisted preferences.
+ *
+ * This property is used to store user preferences across page reloads and is
+ * currently used by the block editor for preferences like 'fullscreenMode' and
+ * 'fixedToolbar'.
+ *
+ * @since 6.1.0
+ * @access private
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ */
+function wp_register_persisted_preferences_meta() {
+	// Create a meta key that incorporates the blog prefix so that each site
+	// on a multisite can have distinct user preferences.
```

Comment:
```suggestion
	/*
	 * Create a meta key that incorporates the blog prefix so that each site
	 * on a multisite can have distinct user preferences.
	 */
```
This should use the correct multiline comment format. [Ref](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-2-multi-line-comments)